libtenzir/builtins/operators/to_write_save.cpp

```diff
@@ -333,13 +422,14 @@ class write_and_save_operator final
   }
 
   auto name() const -> std::string override {
-    return "internal-write-save";
+    return fmt::format("internal-write-save-{}-progress",
+                       ShowProgress == show_progress::yes ? "with" : "without");
```

Comment:
Do we actually want/need the `internal-` prefix here? It's going to be visible to the user via error messages and `--dump-ast`, so it's not really internal.

Reply:
I don't consider `--dump-ast` to be user-facing, so for that I wouldn't wanna do it.

Do you have a suggestion for an alternative name that makes more sense? It can't be `write-save`, and it also can't be `to`, as those names are taken already.

Reply:
Just `write-save-{}-progress`, without the `internal-` prefix?